test/no-array-reduce.mjs

```diff
@@ -93,12 +93,21 @@ test({
 		// Second argument is not a function
 		...notFunctionTypes.map(data => `Array.prototype.reduce.call(foo, ${data})`),
 
-	].flatMap(code => [code, code.replace('reduce', 'reduceRight')]),
-	invalid: [
+		// Option: allowSimpleOperations
 		'arr.reduce((total, item) => total + item)',
```

Comment:
```suggestion
		'array.reduce((total, item) => total + item)',
```

(I know the existing code uses `arr`, but better to use something readable going forward)

Reply:
You want to keep the diff clean and just update the new lines added or would you like me to make the rest more readable as well?

Reply:
Make the rest of them readable too.

Reply:
Addressed in this [commit](https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1418/commits/d0a00c0337e90fab916c275c436f570db44d3a08).